src/components/FAQSection/FAQSection.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from "react";
+import { FAQEntry } from "../";
+import styled from "styled-components";
+
+const FAQTable = styled.table`
+  td,
+  th {
+    width: 30%;
+    text-align: left;
+    vertical-align: top;
+  }
+  th {
+    padding-bottom: 50px;
+  }
+  td {
+    padding-bottom: 50px;
+    padding-right: 50px;
+  }
+`;
+
+const FAQHeader = styled.div`
+  margin-bottom: 15px;
+  font-family: "Roboto", sans-serif;
+  font-size: 48px;
+  font-weight: 500;
+  color: #3cbfce;
+`;
+
+const FAQHeaderCopy = styled.div`
+  font-family: "Roboto Slab", serif;
+  font-size: 32px;
+  font-weight: 400;
+`;
+
+const FAQInfo = [
+  {
+    title: "Where is it?",
+    body:
+      "George Sherman Union, 2nd Floor in Metcalf Hall at 775 Commonwealth Avenue, Boston, MA, 02215."
+  },
+  {
+    title: "Who’s coming?",
+    body:
+      "We expect 400 to 450 hackers from all over North America to attend BostonHacks."
+  },
+  {},
+  {},
+  {
+    title: "Who can attend?",
+    body:
+      "All students are welcome. Unfortunately, we will not be able to accommodate students under 18 this year (unless you are a BU student)."
+  },
+  {
+    title: "Do I need experience?",
+    body:
+      "No experience necessary. We will have plenty of mentors and resources, along with several workshops targeted for beginners. Come learn and experience your first hackathon at BostonHacks!"
+  },
+  {
+    title: "Are there any rules?",
+    body:
+      "We want to ensure a positive experience for all participants. We encourage you to read the Code of Conduct."
+  },
+  {
+    title: "What should I bring?",
+    body:
+      "You should bring your laptop, phone, chargers, toiletries, and optionally a sleeping bag. All food and other accommodations will be taken care of."
+  },
+  {},
```

Comment:
What are these lone brackets doing? (not familiar with react)

Reply:
These are the blank spots on the table for the FAQ, since the design has some parts of the FAQ that don't require text

Reply:
This needs to play nice on mobile. Can this be reworked @nampham148 ?

Reply:
Maybe we can use an [existing grid system](https://github.com/jxnblk/grid-styled) instead of a table here? Would make it easy to collapse to mobile. Also perhaps hide the filler blocks on mobile? (CSS class etc.)

Reply:
Let me look into that